src/App/components/FormComponents/Radio.js

```diff
@@ -15,25 +15,37 @@ const Radio = ({ id, checked, disabled, label, group, groupTitle, options, name,
         <>
             {group ?
                 <form>
-                    <fieldset className={`radio-group${className ? ` ${className}` : ""}`} disabled={disabled}>
-                        <legend>
-                            {groupTitle}
-                            {tooltip && <>
-                                {"\n"}<i className="material-icons help-icon" data-tooltip="Some informative text" data-tooltip-position="top">{"\n"}
-                                help_outline{"\n"}</i>
-                            </>}{"\n"}
-                        </legend>
+                    <fieldset className={`radio-group${errorMessage ? " has-error" : ""}${className ? ` ${className}` : ""}`} disabled={disabled}>{"\n"}
+                        <label className="radio-label">
+                            {groupTitle} {optional && <span>(optional)</span>}
+                        </label>{"\n"}
                         {options.map(({ label, id, checked }, i) => <div className="radio" key={i}>{"\n"}
                             <input {...attrs} id={id} defaultChecked={checked}/>{"\n"}
                             <label htmlFor={id}>{label}</label>{"\n"}
                         </div>)}
+                        {errorMessage ? <div className="help-block">{errorMessage}</div> : null}
+                        {hintText && <div className="hint-text">{hintText}</div>}
+                        {expandingHintText && 
+                        <label className="help-block-expander">
```

Comment:
Same as the checkbox, style hint text and expander

![image](https://user-images.githubusercontent.com/44967773/164174623-1057dfe2-fb07-43ba-941a-9276be94efc0.png)